pkgs/applications/video/mpv/scripts/autosub.nix

```
@@ -0,0 +1,34 @@
+{
+  lib,
+  buildLua,
+  fetchFromGitHub,
+  unstableGitUpdater,
+  python3,
```

```suggestion
  python3Packages,
```

This is a bit of dark magic to explain, just believe me here: `python3.pkgs` **is not** equivalent to `python3Packages`.

Nixpkgs is crafted so that the typical `mkDerivation` is transparent to cross-compilation.
However, this dark magic fails in expressions like `python3.pkgs`, while working in `python3Packages`.